### ISSUE TITLE
Parental controls

### ISSUE THREAD

**Issue by Cigaras** [Cigaras/IPTV.bundle#95]
Ability to set a password that must be entered before plug-in can be accessed.

Just putting this out here, if many users need that or I will have too much spare time on my hands.

**Comment by freddysmith1** [comment]
ill put my hand up for wanting this one,   Have many dirt pervs who use my plex that would like some adult channels lol

**Comment by Cigaras** [comment]
I examined the possibilities and they are quite limited.

First of all there is no way for a user to enter a discreet password, only possible input is search box ([`InputDirectoryObject`](https://forums.plex.tv/discussion/90168/inputdirectoryobject-on-plex-web)) that is completely exposed to passing eye.

Furthermore, `InputDirectoryObject` does not work on current Plex Web interface and many other players, and only workaround I found is [DumbKeyboard](https://forums.plex.tv/discussion/194503/dumbkeyboard-dumbprefs-my-workaround-for-clients-lacking-inputdirectoryobjects-and-prefsobjects) by [coryo](https://github.com/coryo), which is a great add-on but inputting stuff with it is very tedious and requires [`Elevated`](https://docs.google.com/document/d/1MyhhTsg5xdDD5LRbOZ5x5QxkmEyXv6rB-22Z7tdpy34/edit) flag which I do not want in my plug-in.

In short, I am not going to implement this feature at this state, maybe sometime later when Plex improves its Framework.
